pkg/utils/file_filter.go

```diff
@@ -117,6 +123,33 @@ func parseIgnoreFile(content []byte, filePath string) (ignores []string) {
 	return ignores
 }
 
+// parseDotSnykFile builds a list of glob patterns from a given .snyk style file
+func parseDotSnykFile(content []byte, filePath string) []string {
+	type DotSnykRules struct {
+		Exclude struct {
+			Code   []string `yaml:"code"`
+			Global []string `yaml:"global"`
+		} `yaml:"exclude"`
+	}
+
+	var rules DotSnykRules
+	err := yaml.Unmarshal(content, &rules)
+	if err != nil {
+		fmt.Printf("parse .snyk failed: %v", err)
```

Comment:
**issue(blocking):** Rather than outputting to fmt.Printf, can we pass a logger through to this method.